lib/Doctrine/MongoDB/Aggregation/Stage/AbstractBucket.php

```diff
@@ -57,6 +57,21 @@ public function getExpression()
     }
 
     /**
+     * Converts an expression object into an array, recursing into nested items
+     *
+     * For expression objects, it calls getExpression on the expression object.
+     * For arrays, it recursively calls itself for each array item. Other values
+     * are returned directly.
```

Comment:
Does this comment still apply? I wonder if it makes more sense to explain the purpose of this method as an extendable hook, rather than repeating the documentation for the method it proxies.